pages/polling/[poll-hash].tsx

```diff
@@ -163,9 +174,19 @@ const PollView = ({ poll, polls: prefetchedPolls }: { poll: Poll; polls: Poll[]
                     </Text>
                     <CountdownTimer key={poll.multiHash} endText="Poll ended" endDate={poll.endDate} />
                   </Flex>
-                  <Heading mt="2" mb="3" sx={{ fontSize: [5, 6] }}>
+                  <Heading mt="2" mb="2" sx={{ fontSize: [5, 6] }}>
```

Comment:
I know you didn't write this but we should use `variant="smallHeading"` instead of `fontSize: [5, 6]`. We'll fix it in the redesign :P